stream/client.py

```diff
@@ -133,6 +133,8 @@ def get_user_agent(self):
 
     def _parse_response(self, response):
         try:
+            if response.encoding is None:
```

Comment:
Here we're naively forcing 'utf-8' which will break in future if the API start returning responses with a different content-type header (that doesn't allow requests to "guess" the encoding).

An improvement would be:
```
if response.encoding is None and 'application/json' in response.headers.get('content-type'):
  response.encoding = 'utf-8'
```

Another option to check if 'requests' exposes the logic / function it makes use of to guess encoding for response.json().